src/ui/layout/search/GrandSearch.vue

```diff
@@ -3,10 +3,12 @@
     ref="GrandSearch"
     aria-label="OpenMCT Search"
     class="c-gsearch"
+    role="searchbox"
 >
     <search
         ref="shell-search"
         class="c-gsearch__input"
+        tabindex="0"
```

Comment:
Since we know that the searchbar should always be the first "tabbable" field, we can explicitly calculate the tabindex to 0